src/Payments.sol

```diff
@@ -181,7 +181,7 @@ contract Payments is ReentrancyGuard {
         string note;
     }
 
-    constructor() ReentrancyGuard() {
```

Comment:
The lint error for this line was:
```
error: modifier-style base constructor call without arguments
   --> /Users/will/projects/filecoin-pay/src/Payments.sol:184:19
    |
184 |     constructor() ReentrancyGuard() {
    |                   ^^^^^^^^^^^^^^^^^
    |
```
This is because the superclass constructor is called implicitly if it has no arguments.